test/unit/math/rev/mat/fun/mdivide_left_tri_test.cpp

```diff
@@ -355,6 +356,14 @@ TEST(AgradRevMatrix,mdivide_left_tri_upper_grad_vd) {
     }
   }
 }
+
+TEST(AgradRevMatrix, check_varis_on_stack) {
+  stan::math::matrix_v A(2,2);
+  A << 2.0, 0.0,
+    5.0, 7.0;
+  test::check_varis_on_stack(stan::math::mdivide_left_tri<Eigen::Lower>(A, A));
+}
+
 // // FIXME:  Fails in g++ 4.2 -- can't find agrad version of mdivide_left_tri
```

Comment:
We don't support g++ 4.2 --- too early.  Also, in general, we to change `FIXME:` to `TODO(name):`